src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -74,6 +74,6 @@ WebexMeeting.defaultProps = {
   // eslint-disable-next-line no-confusing-arrow
   controls: (isActive) =>
     isActive
-      ? ['mute-audio', 'mute-video', 'share-screen', 'leave-meeting']
+      ? ['mute-audio', 'mute-video', 'share-screen', 'leave-meeting', 'member-roster']
```

Comment:
Here should be `participant-list`. It's an icon. https://momentum-ui.com/icons

Reply:
This array contains the names of the controls available in adapter not the names of the icons, the icon name is added inside de control